package.json

```diff
@@ -19,6 +19,7 @@
   "license": "MIT",
   "dependencies": {
     "@lhci/server": "^0.4.1",
+    "axios": "^0.19.2",
```

Comment:
So this is being used in client right, then this should actually go into `client/package.json`. Our repo is divided into 2 sub-repos (client and server) so client directory itself is like a repo

Reply:
Hmm. I could've sworn i did an npm uninstall there. Oh well, I'll fix it